src/signals/incident-management/containers/DefaultTextsAdmin/components/DefaultTextsForm/index.js

```diff
@@ -228,22 +212,20 @@ const DefaultTextsForm =({
                     icon={<ChevronDown />}
                     onClick={e => changeOrdering(e, index, 'down')}
                   />
-                </StyledColumn>
-              </Row>
+                </StyledRightColumn>
+              </div>
             ))}
 
-            <Row>
-              <StyledColumn span={12}>
-                <Button
-                  data-testid="defaultTextFormSubmitButton"
-                  variant="secondary"
-                  type="submit"
-                  disabled={invalid}
-                >
+            <div>
```

Comment:
Is wrapping the button in a div necessary? The preceding element is already a block-level element.

Reply:
gedaan